Core/SubscriptionManager/src/main/java/eu/neclab/ngsildbroker/subscriptionmanager/service/BaseNotificationHandler.java

```diff
@@ -582,10 +582,17 @@ public Part getPart(String name) throws IOException, ServletException {
             
 			dataArray.add(jsonTree.get("data"));
 			((ObjectNode) jsonTree).set("data", dataArray);
-			
+			Map<String, String> map = new HashMap<String, String>();
+			if (context != null && !context.isEmpty()) {
+				map.put(NGSIConstants.ACCEPTED_LINK, context.get(0).toString());
+			}
+			map.put(NGSIConstants.CONTENT_TYPE, acceptHeader);
+			Map<String, Object> mapObj = new HashMap<String, Object>();
+			mapObj.put(NGSIConstants.METADATA, map);
+			mapObj.put(NGSIConstants.BODY, jsonTree);
 			BodyBuilder builder = ResponseEntity.status(HttpStatus.ACCEPTED);
 
-			return builder.headers(temp.getHeaders()).body(objectMapper.writeValueAsBytes(jsonTree));
+			return builder.headers(temp.getHeaders()).body(objectMapper.writeValueAsBytes(mapObj));
```

Comment:
Hi,
please revert this part here. This encapsulates all notifications in the mqtt format. Which shouldn't be the case. Only the mqtt ones need that. I've moved that, on the last merge, into the mqttnotificationhandler. So you can just remove it here again or better just update from dev again.